stm32-modules/heater-shaker/include/heater-shaker/messages.hpp

```diff
@@ -84,6 +88,12 @@ struct SetPowerTestMessage {
     double power;
 };
 
+struct SetSerialNumberMessage {
+    uint32_t id;
+    static constexpr std::size_t serial_number_length = 8;
```

Comment:
we might want to derive this length or store it from some single source of truth at least for all the template arguments